test/run-helm-tests.sh

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+K8S_VERSION=${KUBERNETES_VERSION:-v1.20.2}
+VERSION=helm-test make image
+wget https://github.com/kubernetes-sigs/kind/releases/download/v0.10.0/kind-linux-amd64
+chmod +x kind-linux-amd64
+mv kind-linux-amd64 kind
+export PATH=$PATH:$PWD
+kind create cluster --image kindest/node:"${K8S_VERSION}" --config=./hack/kind_config.yaml
+kind load docker-image descheduler:helm-test
+helm install descheduler-ci --set image.repository=descheduler,image.tag=helm-test --namespace kube-system ./charts/descheduler
```

Comment:
Last change for this shell script ...

We want to be able to test different versions of the descheduler helm chart. For example a released version of the chart with released version of the container image. The CLI options you have are correct, and we want these to be the defaults.

So `--set image.repository=descheduler` needs a variable that can be overridden, `--set image.tag=helm-test` needs a variable. and the last parameters `./charts/descheduler` needs a variable.

Let me know if you need more details. Thanks!

Reply:
Updated with the latest push.